docs/bsp.md

```diff
@@ -829,7 +829,7 @@ trait CompileParams {
   
   /** A unique identifier generated by the client to identify this request.
     * The server may include this id in triggered notifications or responses. */
```

Comment:
s/or responses//

JSON-RPC responses already include an ID matching the request. The `originID` is only relevant for notifications.